src/redis/fixed_cache_impl.go

```diff
@@ -32,119 +32,131 @@ type fixedRateLimitCacheImpl struct {
 	baseRateLimiter                    *limiter.BaseRateLimiter
 }
 
-func pipelineAppend(client Client, pipeline *Pipeline, key string, hitsAddend uint32, result *uint32, expirationSeconds int64) {
+func pipelineAppend(client Client, pipeline *Pipeline, key string, hitsAddend uint64, result *uint64, expirationSeconds int64) {
 	*pipeline = client.PipeAppend(*pipeline, result, "INCRBY", key, hitsAddend)
 	*pipeline = client.PipeAppend(*pipeline, nil, "EXPIRE", key, expirationSeconds)
 }
 
-func pipelineAppendtoGet(client Client, pipeline *Pipeline, key string, result *uint32) {
+func pipelineAppendtoGet(client Client, pipeline *Pipeline, key string, result *uint64) {
 	*pipeline = client.PipeAppend(*pipeline, result, "GET", key)
 }
 
+func (this *fixedRateLimitCacheImpl) getHitsAddend(hitsAddend uint64, isCacheKeyOverlimit, isCacheKeyNearlimit,
```

Comment:
do you want to add unit tests for this function?

Reply:
Of cource. I will add some unit tests for the whole feature.